sortinghat/cmd/load.py

```diff
@@ -423,11 +423,16 @@ def __load_identities(self, identities, uuid, verbose):
     def __load_profile(self, profile, uuid, verbose):
         """Create a new profile when the unique identity does not have any."""
 
+        def is_empty_profile(prf):
```

Comment:
why is_empty_profile is inside __load_profile?

Reply:
It's a private function for that method. Only used there and I don't think other methods will need it.